aws-rds-dbsubnetgroup/src/main/java/software/amazon/rds/dbsubnetgroup/BaseHandlerStd.java

```diff
@@ -56,6 +59,31 @@ protected boolean isStabilized(final ResourceModel model, final ProxyClient<RdsC
       return status.equals(DB_SUBNET_GROUP_STATUS_COMPLETE);
     }
 
+    public ProgressEvent<ResourceModel, CallbackContext> handleException(final Exception e) {
+        if (
```

Comment:
I am not sure if this is the formatting we use. I thought we had something like this

```
if (e instanceof Blah) {
    return blah;
}

```
expression in if or else if - in one line
4 spaces indentation.

Reply:
Done